Makefile

```diff
@@ -65,12 +65,23 @@ fill-cache-for-e2e-regression: oasis-indexer
 
 # Run the api tests locally, assuming the environment is set up with an oasis-node that is 
 # accessible as specified in the config file.
-test-e2e-regression: export TZ=UTC
 test-e2e-regression: oasis-indexer
-	./oasis-indexer --config tests/e2e_regression/e2e_config.yml analyze
+	TZ=UTC ./oasis-indexer --config tests/e2e_regression/e2e_config.yml analyze
```

Comment:
Unrelated. No-op change, just makes the command easier to copy-paste out of the Makefile

Reply:
I think we actually need to move this to the line that instantiates the api server in `run.sh`, since the timestamp is formatted by the api server.